crates/vizia_core/src/context/draw.rs

```diff
@@ -100,9 +101,9 @@ impl<'a> DrawContext<'a> {
         self.cache.get_clip_region(self.current)
     }
 
-    /// Returns the name of the default font.
-    pub fn default_font(&self) -> &str {
-        &self.style.default_font
+    /// Returns the lookup pattern to pick the default font.
+    pub fn default_font(&self) -> Vec<FamilyOwned> {
```

Comment:
Maybe this should return by reference to avoid the clone? I checked and this isn't actually used anywhere so I don't know what the intended use would be.